tests/global_config.py

```diff
@@ -204,12 +205,12 @@ def _get_default_storage_class(sc_list):
 ]
 
 data_import_cron_matrix = [
-    {"centos-stream9": {"instance_type": "u1.medium", "preference": "centos.stream9"}},
-    {"centos-stream10": {"instance_type": "u1.medium", "preference": "centos.stream10"}},
-    {"fedora": {"instance_type": "u1.medium", "preference": "fedora"}},
-    {"rhel8": {"instance_type": "u1.medium", "preference": "rhel.8"}},
-    {"rhel9": {"instance_type": "u1.medium", "preference": "rhel.9"}},
-    {"rhel10": {"instance_type": "u1.medium", "preference": "rhel.10"}},
+    {"centos-stream9": {"instance_type": U1_MEDIUM_STR, "preference": "centos.stream9"}},
+    {"centos-stream10": {"instance_type": U1_MEDIUM_STR, "preference": "centos.stream10"}},
+    {"fedora": {"instance_type": U1_MEDIUM_STR, "preference": "fedora"}},
+    {"rhel8": {"instance_type": U1_MEDIUM_STR, "preference": "rhel.8"}},
+    {"rhel9": {"instance_type": U1_MEDIUM_STR, "preference": "rhel.9"}},
+    {"rhel10": {"instance_type": U1_MEDIUM_STR, "preference": "rhel.10"}},
```

Comment:
can we save and reuse preferences as we are using then later in instance_type_rhel_os_matrix

Reply:
I only updated the ones in here for this PR, lets do it globally for the entire repository later - https://issues.redhat.com/browse/CNV-65113